### PR TITLE
Keyboard navigation keys handling

### PR DESCRIPTION
**Pull request by xperiandri**
@mfkl, could you try this branch?

### CODE REVIEW

File: app/VLC.Universal/VLC.Universal.csproj [hunk 1]
```diff
@@ -1647,6 +1647,7 @@
     <Warning Text="$(MSBuildProjectFile) is Multilingual build enabled, but the Multilingual App Toolkit is unavailable during the build. If building with Visual Studio, please check to ensure that toolkit is properly installed." />
   </Target>
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it.
```
Comment (mfkl) [inline app/VLC.Universal/VLC.Universal.csproj]: This extra line you probably did not want to commit breaks the csproj (and project loading in VS).

**Comment by mfkl** [comment]
Can you rebase on master please?

**Comment by xperiandri** [comment]
Done

**Comment by mfkl** [comment]
Sorry for the delay dude, I'll have a look next week.

**Comment by mfkl** [comment]
Ok I tested this, looks good thanks.

Can you remove your .gitignore changes from the PR please?

It conflicts with master since we did update it there.

**Comment by xperiandri** [comment]
Don't you want to mix gitignore update into this PR?
Or you don't like my gitignore? I've just deleted yours and added default by Visual Studio. It is simply covers more cases.

**Comment by xperiandri** [comment]
> Ok I tested this, looks good thanks.
Thanks

**Comment by mfkl** [comment]
> Don't you want to mix gitignore update into this PR?

No I don't. Let's keep this PR focused on what its about.

> It is simply covers more cases.

It also lacks some stuff that are present in the current one (like thumbs.db). I'd rather manage this myself instead of letting VS do that for me, for now.

**Comment by xperiandri** [comment]
I updated PR.
I need that 4 ignores.
I wonder why don't you ignore .vs

**Comment by mfkl** [comment]
Thanks for your code update.

There seems to be small regression though. Volume up/down shortcuts do not work anymore for me (music/video pages).

**Comment by xperiandri** [comment]
Will you have a look?
I still cannot launch playback

**Comment by mfkl** [comment]
>Will you have a look?

Yes, I will next week and let you know.

> I still cannot launch playback

We need to get this fixed for you. If you're on latest master with no local changes, I'm pretty sure it comes from a bad nightly build. Those can be pretty unstable sometimes, so feel free to grab some older ones and try.

**Comment by mfkl** [comment]
Merged, thanks.

**Pull request closed without merging**